drivers/volume/portworx/portworx.go

```diff
@@ -638,7 +638,7 @@ func (d *portworx) updateNode(n *node.Node, pxNodes []*api.StorageNode) error {
 					// TODO: PTX-2445 Replace isMetadataNode API call with SDK call
 					isMetadataNode, err := d.isMetadataNode(*n, address)
 					if err != nil {
-						log.Warnf("can not check if %v is metadata node", *n)
+						log.Warnf("can not check if %v is metadata node,err: %v", *n, err)
```

Comment:
Can we improve the logging a bit and make it less complex and more readable. Don't need to give the whole node block, I think just the name will be more readable for debugging purposes, you could maybe add `n.NodeId` if you think its necessary
`log.Warnf("Unable to check if node [%s] is metadata node, Err: %v", *n.Name, err)`

Reply:
Also why are we not returning here? Why warning? I think we should return on error, no?

Reply:
Fixed the node name logging

it is update method, running as pre-check, it fails to check the metadatanode value  sometime and ticket PTX-2445 is raised for the same. Do not want to block the tests due to this. Will address this along with PTX-2445